admin/services/data-access/setting.data.service.js

```diff
@@ -1,11 +1,24 @@
 'use strict'
 
 const Setting = require('../../models/setting')
+const sqlService = require('./sql.service')
+const TYPES = require('tedious').TYPES
 
 const settingDataService = {}
 
-settingDataService.findOne = async function (options) {
-  return Setting.findOne(options).lean().exec()
+settingDataService.findOne = async function () {
+  const sql = 'SELECT * FROM Settings WHERE id=@id'
+  const params = [ {
+    name: 'id',
+    value: 1,
+    type: TYPES.Int
+  }]
+
+  const result = await sqlService.query(sql, params)
+  if (result && result.length === 1) {
```

Comment:
The expectation is that findOne returns the first row, no matter how many results there are, or what it's id is.

Something more like `SELECT TOP 1 from $TABLE`

Reply:
@jon-shipley true.  But the settings table is an exception due to the fact that there should only ever be one row.  This is why i omitted the auto increment on the primary key.
Happy to change it, as it doesnt necessarily have to row with `id=1`
Thoughts?